setup.cfg

```diff
@@ -52,3 +53,6 @@ twine.registered_commands =
     register = twine.commands.register:main
 console_scripts =
     twine = twine.__main__:main
+
+[options.package_data]
+twine = py.typed
```

Comment:
Instead of this, use `include_package_data=True` in the `[options]`. This latter approach works in the general case and avoids redundantly "adding" the files to the SCM and then "adding" them to the package.

Reply:
Done in https://github.com/pypa/twine/pull/618/commits/ad918211ee42eb2c437c4ef7f42372a595715d89.